src/test/java/me/karakelley/tictactoe/IOTest.java

```diff
@@ -31,14 +29,26 @@ public void testGetInput() {
 
     IO getIO = new IO(stdout, in);
     assertEquals("", getIO.getInput());
+    System.setIn(stdin);
   }
 
   @Test
   public void testDisplay() {
     PrintStream originalOut = System.out;
     System.setOut(stdout);
     IO.display("Test");
+
     assertEquals("Test", outContent.toString());
     System.setOut(originalOut);
   }
+
+  @Test
+  public void testAnyKey() throws IOException, InterruptedException {
+    ByteArrayInputStream input = new ByteArrayInputStream("T".getBytes());
+    System.setIn(input);
```

Comment:
Why do you need to `setIn`. Aren't you injecting your own stream into the constructor so you don't need to do this?